src/components/Notifications/utils.tsx

```diff
@@ -81,6 +86,8 @@ export const NotificationComponentByType: DecentralandNotificationComponentByTyp
     [NotificationType.LAND_RENTED]: LandRentedNotification,
     [NotificationType.LAND_RENTAL_ENDED]: LandRentalEndedNotification,
     [NotificationType.REWARD_ASSIGNED]: RewardAssignedNotification,
+    [NotificationType.REWARD_IN_PROGRESS]: null,
+    [NotificationType.BADGE_GRANTED]: null,
```

Comment:
What do you think about this?
```suggestion
    [NotificationType.REWARD_IN_PROGRESS]: GovernanceVotedOnBehalfNotification,
    [NotificationType.BADGE_GRANTED]: GovernanceWhaleVoteNotification,
```

Reply:
Added these as needed for the DCLNotification type.